pkg/pillar/cmd/zedagent/zedagent.go

```diff
@@ -1924,3 +1931,9 @@ func getDeferredPriorityFunctions() []zedcloud.TypePriorityCheckFunction {
 	})
 	return functions
 }
+
+func runHandleDeferred(ps *pubsub.PubSub, change time.Time) {
+	start := time.Now()
+	zedcloud.HandleDeferred(zedcloudCtx, change, 100*time.Millisecond, false)
+	ps.CheckMaxTimeTopic(agentName, "deferredChan", start, warningTime, errorTime)
+}
```

Comment:
I'm not a fan of taking a few repeated lines of code and making them a function. It is actually better to repeat them because it makes it more clear and uniform to the reader.

IMHO a function should provide an abstraction which can be understood and fit in with the larger pattern of abstractions in the code. This one doesn't